docs/content/040-adoption-guides/040-nexus-with-nextjs.md

```diff
@@ -0,0 +1,328 @@
+---
+title: Nexus With Next.js
+---
+
+## Nexus in a Next.js Project
+
+Next.js comes with a feature called [API Routes](https://nextjs.org/docs/api-routes/introduction). With API Routes, we can easily create dedicated endpoints to use in our Next.js applications. These endpoints can then be deployed as serverless functions.
+
+In this guide, we'll see how to work with Next.js API Routes to serve a Nexus Schema GraphQL API and then consume it from the front end.
+
+## Getting Started
+
+Let's start by creating a new Next.js application. If you already have a Next.js app, you can ignore this step.
+
+```bash
+npx create-next-app
+# Provide a name for your app
+```
+
+After installation, we need to opt-in to use TypeScript. To do this, let's create a `tsconfig.json` file at the project root. Next.js will take care of populating this file for us.
+
+```bash
+touch tsconfig.json
+```
+
+Next, install `typescript` and `@types/react`.
+
+```bash
+npm i -D typescript @types/react
+```
+
+To finish out the TypeScript setup, run the app in development mode. This will populate the `tsconfig,json` file.
+
+```bash
+npm run dev
+```
+
+For more information on using TypeScript in Next.js, check out [the docs](https://nextjs.org/docs/basic-features/typescript).
+
+With the Next.js project in place, we now need to install the other dependencies we'll need.
+
+In addition to Nexus Schema, we'll need a server to run the GraphQL API. Nexus Schema works with a range of GraphQL servers. We'll use Apollo Server and opt for the `apollo-server-micro` variety as it is well-suited to serverless environments.
```

Comment:
We're moving back toward just saying "Nexus". Package will be renamed to `nexus` in 1.0.